devito/passes/clusters/temporal.py

```diff
@@ -0,0 +1,78 @@
+from devito.passes.clusters.utils import cluster_pass
+from devito.ir.support import IterationSpace, SEQUENTIAL
+from devito.ir.support.space import Interval
+from devito.symbolics import xreplace_indices
+from devito.logger import warning
+
+__all__ = ['skewing']
+
+
+@cluster_pass
+def skewing(cluster, *args):
+
+    """
+    Skew the accesses along a SEQUENTIAL Dimension.
+    Example:
+
+    Transform
+
+    for i = 2, n-1
+        for j = 2, m-1
+            a[i,j] = (a[a-1,j] + a[i,j-1] + a[i+1,j] + a[i,j+1]) / 4
+        end for
+    end for
+
+    to
+
+    for i = 2, n-1
+        for j = 2+i, m-1+i
+            a[i,j-i] = (a[a-1,j-i] + a[i,j-1-i] + a[i+1,j-i] + a[i,j+1-i]) / 4
+        end for
+    end for
+    """
+    processed = []
+    # What dimensions do we target?
+    # a) SEQUENTIAL Dimensions
+    skew_dims = []
```

Comment:
use a list comprehension, not four lines. That's the pythonic way

Reply:
ok